### PR TITLE
Improve the yellow traffic light recognition 

### PR DESCRIPTION
**Pull request by lchojnack**
Change the camera image resolution from 1280x720 to 1920x1080 and the camera focal length from 20mm to 35mm

The changes improve the yellow traffic light recognition, mentioned in https://github.com/tier4/AWSIM/issues/10

Before changes

https://user-images.githubusercontent.com/8416175/210326803-17767916-c0dd-4f96-beee-7de3f1b87c46.mp4

After changes

https://user-images.githubusercontent.com/8416175/210326985-e856d900-0741-4267-af70-36ddfc0836dc.mp4

### CODE REVIEW

**Review by shmpwk — APPROVED** [review]
LGTM, waiting for @mackierx111 review.

**Comment by mackierx111** [comment]
LGTM. 
However, we will merge this pull request after `awsim-stable` branch reflects the parameters regarding Autoware's traffic light vibration.
https://github.com/autowarefoundation/autoware.universe/pull/3189

**Comment by shmpwk** [comment]
Merge this because `awsim-stable` branch reflects this change.

**Pull request merged**
